docs/authmate.md

```diff
@@ -127,16 +184,18 @@ it will be auto-generated with values:
 }
 ```
 
-With session token, there is 3 options: 
-* append `--session-token` parameter with your custom rules in json format (as a string or file path, see an example below)
-
-**NB!** To create buckets in NeoFS it's necessary to have session tokens with `PUT` and `SETEACL` permissions, that's why 
-the authmate creates a `SETEACL` session token automatically in case when a user specified the token rule with `PUT` and 
-forgot about the rule with `SETEACL`. 
+### Session tokens
 
-* append `--session-token` parameter with the value `none` -- no session token will be created
-* skip the parameter and `authmate` will create and put session tokens with default rules:
+With session token, there are 3 options: 
+1. append `--session-token` parameter with your custom rules in json format (as a string or file path). E.g.:
```

Comment:
It seems `--session-tokens` is more appropriate name.Any thoughts?

Reply:
Seems good. I would rename CLI argument in separate commit \ PR, though.

/cc @realloc